forch/forch_proxy.py

```diff
@@ -24,7 +24,7 @@ def __init__(self, proxy_config):
     def start(self):
         """Start proxy server"""
         self._register_pages()
-        self._proxy_server = HttpServer(self._proxy_port)
+        self._proxy_server = HttpServer(self._proxy_port, content_type='text/plain')
```

Comment:
add a comment here saying that forch_proxy is only currently used for varz so this server can be hard-wired to text/plain

Reply:
Renamed proxy instance variable instead to clarify that the instance is only used for varz